doc/tutorial/Server.lhs

```diff
@@ -8,7 +8,15 @@ type. Can we have a webservice already?
 Equipped with some basic knowledge about the way we represent APIs, let's now
 write our first webservice.
 
-The source for this tutorial section is a literate haskell file, so first we
+## Setting up our project
+
+Don't forget to look at the [cabal file included with this example project](doc/tutorial/tutorial.cabal). You can run the main function provided in the following examples within the `Server` module without using a separate executable.
+
+If you're using Stack, then in your `stack.yaml` you may want to specify version 0.11.1.1 for `aeson`, and version 0.5 for `servant` and `servant-server` in your `extra-deps`.
```

Comment:
@bitemyapp: Ah, that makes sense.

Maybe we should encourage people to checkout the `servant` repo and play around with the tutorial in the repo. And explain in detail how to do that in the `README`. This would have the added benefit that it would make it easier for people to contribute something to the tutorial.

On the other hand it's maybe a better learning experience if you actually set up files and write code yourself while reading the tutorial.

@bitemyapp: I can't speak for other maintainers, but I'd be happy to leave that decision up to you. Since you're the one working on this.

Generally it would be nice to have a very minimal `servant` project to get people started. Including cabal file, stack file, test suite, etc.. (This of course has little to do with your PR.)